.dotnet/src/Custom/Chat/ChatCompletionOptions.cs

```diff
@@ -144,6 +145,32 @@ public ChatCompletionOptions()
     [Experimental("OPENAI001")]
     public long? Seed { get; set; }
 
+    // CUSTOM: Hide deprecated max_tokens and reroute to newer max_completion_tokens.
+    [CodeGenMember("MaxTokens")]
+    internal int? _deprecatedMaxTokens { get; set; }
```

Comment:
Thew "max_completion_tokens" works with all models, right? Including the ancient ones like gpt-3.5-turbo?

Reply:
Yes, thankfully! On OpenAI, it's "safe" (and encouraged, I believe) to always use the new `max_completion_tokens`.